crates/settings/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "settings"
+description = "Source of truth for Air settings"
+version = "0.0.0"
+publish = false
+authors.workspace = true
+categories.workspace = true
+edition.workspace = true
+homepage.workspace = true
+keywords.workspace = true
+license.workspace = true
+repository.workspace = true
+
+[features]
+biome = ["dep:biome_formatter"]
+serde = ["dep:serde"]
+
+[dependencies]
+biome_formatter = { workspace = true, optional = true }
+serde = { workspace = true, optional = true }
```

Comment:
Note that serde support and biome conversion support are optional

Probably overkill but it feels quite "professional" to do it this way